packages/block-library/src/template-part/edit/advanced-controls.js

```diff
@@ -53,13 +78,41 @@ export function TemplatePartAdvancedControls( {
 						onFocus={ ( event ) => event.target.select() }
 					/>
 
-					<SelectControl
-						label={ __( 'Area' ) }
-						labelPosition="top"
-						options={ AREA_OPTIONS }
-						value={ area }
-						onChange={ setArea }
-					/>
+					<BaseControl
```

Comment:
I wonder if `RadioGroup`/`Radio` components would be able to fit well in this case with wanting the icon buttons?  It seems pretty similar visually and behaviorally to what we are going for.

Reply:
+1 for RadioGroup and Radio components. They seem to fit this particular situation.

Reply:
Thanks for the idea, I will take a look.